doc/user/sql-grammar/generate.sh

```diff
@@ -22,14 +22,14 @@ rm -rf scratch
 
 # Run the railroad diagram generator, using a pinned version from our custom
 # fork.
-docker run --rm -i materialize/rr:0.0.4 -nostyles -svg -noinline -width:600 - < sql-grammar.bnf > diagrams.zip
+docker run --rm -i materialize/rr:v0.0.5 -nostyles -svg -noinline -width:600 - < sql-grammar.bnf > diagrams.zip
 
 # Extract the SVGs we care about and move them into place.
 mkdir scratch
 (
     cd scratch
     unzip -j ../diagrams.zip
-    rm ./diagrams.zip rr-1.62.svg index.html
+    rm ../diagrams.zip Railroad-Diagram-Generator.svg index.html
```

Comment:
* `rm ./diagrams.zip` was executing within the scratch directory, but the zip actually lives in the parent
* I didn't see any `rr-*.svg` files getting generated, but we do have a straggler `Railroad-Diagram-Generator.svg` so I updated that deletion

^ tried these changes on both an M1 mac + Ubuntu scratch machine, so hopefully it works for everyone